spec/json/HotelSearch_v3_swagger_specification.json

```diff
@@ -455,11 +455,11 @@
         },
         "roomQuantity": {
           "description": "number of rooms (1-9)",
-          "example": 1,
-          "type": "integer",
-          "minimum": 1.0,
-          "maximum": 9.0,
-          "format": "int32"
+          "example": "1",
+          "type": "string",
+          "minimum": 1.0, #to be updated
```

Comment:
minimum, maximum and format properties to be removed

Reply:
removal is done. merci!